### ISSUE TITLE
Menu dropdown disappears

### ISSUE THREAD

**Issue by NumerousHats** [backdrop-contrib/mosaic_grid#34]
I have just encountered a very strange problem where Mosaic Grid seems to cause dropdowns in the main site navigation to disappear. Here are the steps I took to reproduce the problem in the backdropcms.org sandbox as I remember them. Hopefully I haven't missed anything critical...

1. Add a field to content type "Page" called "Image gallery" of type "Image" and allowing an unlimited number of values.
2. Hide that field from the default display.
3. Install and enable the Mosaic Grid module.
4. Create a block view of format "Mosaic Grid" with an "Nid" contextual filter with setting "Provide default value" of type "Content ID from URL".
5. Add that block view to the "Content" region of the default layout.
6. Create two new "Page" nodes, say, "Item A" and "Item B". Make "Item A" a top-level entry in the Primary navigation menu, and make "Item B" a child of "Item A". Add some images to the "Image gallery" field of "Item B".

You may need to clear cache and force reload (to clear your browser's cache), but eventually, if you click on either the "About" or the "Item A" menu link, the little dropdown triangle icon on "Item A" will disappear. Strangely enough, the dropdown works normally if you're on the homepage or if you click on "Item B" via the dropdown when it is visible. Disabling Mosaic Grid restores normal behavior.

This is way beyond my skill level to debug, so hopefully somebody can look into this...

**Comment by korontari** [comment]
Hi @NumerousHats and thank you for reporting this! I apologize for delayed response (Christmas eve has its price), I will check this out and see what is wrong with it. Might be some issue with z-index in CSS.

**Comment by NumerousHats** [comment]
Since things have been a bit silent here, I've gone ahead and tried to debug. It seems that the dropdown disappears only if you are on a page that has no images in the image gallery field.

In that case, the console shows `Uncaught TypeError: Window.getComputedStyle: Argument 1 is not an object.` which I believe is coming from `jquery.flex-images.min.js`. I have a sneaky feeling flex-images is looking for an image to get a size of, but fails to find one because there are none.

**Comment by NumerousHats** [comment]
Yes. Checking to see if a `mosaic-grid-item` actually exists on the page seems to solve the problem. Creating a PR.

**Comment by korontari** [comment]
Hey Michael, 
I've checked your #35 and it worked in my test environment. I have rolled out a release. If the issue reoccurs, feel free to reopen it. Thank you for helping out with this!
